examples/saml-auto-reg/pom.xml

```diff
@@ -0,0 +1,73 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wildfly.wildfly-s2i.examples</groupId>
+    <artifactId>saml</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+
+    <name>WildFly S2I - SAML example</name>
+    <description>An application secured with SAML example</description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <version.maven.war.plugin>3.3.2</version.maven.war.plugin>
+        <version.wildfly>27.0.1.Final</version.wildfly>
+        <version.wildfly.plugin>4.0.0.Final</version.wildfly.plugin>
+        <version.wildfly.cloud.galleon.pack>2.0.0.Final</version.wildfly.cloud.galleon.pack>
+        <jakarta.jakartaee-api.version>10.0.0</jakarta.jakartaee-api.version>
+        <version.keycloak>999.0.0-SNAPSHOT</version.keycloak>
```

Comment:
Just a reminder that we'll need to update the dependency versions here.

As a quick test, just tried making the following updates:

```
<version.wildfly>29.0.0.Final</version.wildfly>
<version.wildfly.plugin>4.2.0.Final</version.wildfly.plugin>
<version.wildfly.cloud.galleon.pack>4.0.0.Final</version.wildfly.cloud.galleon.pack>
<version.keycloak>22.0.1</version.keycloak>
```

With the above updates, I can't seem to get the example working properly on OpenShift. I've created [JBEAP-25308](https://issues.redhat.com/browse/JBEAP-25308) to track this.